docs/Intro/dive/yieldRateOracle.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 5
+description: Yield Rate Oracle
+keywords: [Yield Rate Oracle]
+---
+
+# Chronicle's Yield Rate Oracle
+
+## Overview
+
+In collaboration with [Block Analitica](https://blockanalitica.com/), Chronicle has developed a [Yield Rate Oracle](https://github.com/chronicleprotocol/scribe/blob/main/src/extensions/ScribeLST.sol) that tracks and delivers the value of the ETH staking yield onchain in real-time. This solution is designed to improve the DeFi user experience and reduce governance overhead.
+
+The Yield Rate Oracle is a Schnorr based Oracle with an additional functionality, `getAPR()` - callable by toll'ed (authorized) addresses.
+
+## Purpose
+
+This Oracle provides lending protocols such as [Spark](https://spark.fi/) with a safe and automated way to customize the interest rate of ETH and ETH-backed derivatives. It eliminates the need for manual rate setting and updates, which are common in many decentralized lending protocols.
+
+The liquid staking token interest rate derives from Lido's [stETH](https://etherscan.io/address/0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84) contract.
+
+### Automated Interest Rate Customization
+
+The Oracle tracks the Ethereum Staking Reward Rate and publishes this value onchain. This mechanism enables DeFi platforms to:
```

Comment:
I'm not sure whether we should go with "_the_ Ethereum Staking Reward Rate" as, afaik, there is no "official" rate. 

We could explain why we use lido, ie biggest validator leads to less randomness from block proposal rewards etc.

We should also note that we use the average of the last 2 days, so "real time" is a bit of a tricky term(?)

Reply:
implemented feedback